docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -951,6 +951,12 @@ class A
   @@test = 10
 end
 
+class A
+  class_variable_set(:@@test, 10)
+end
+
+A.class_variable_set(":@@#{name}", 10)
```

Comment:
I think this example will occur a `NameError`.

```suggestion
A.class_variable_set("@@#{name}", 10)
```

Reply:
Good spot. Sometimes comments are the hardest place to get code right first time... :)